src/clr-angular/utils/drag-and-drop/draggable-ghost.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {animate, style, transition, trigger} from "@angular/animations";
+import {Component, ElementRef, HostBinding, Inject} from "@angular/core";
+
+import {DomAdapter} from "../../data/datagrid/render/dom-adapter";
```

Comment:
Why are you using the Datagrid's DomAdapter here? If we end up having to use it everywhere, we should start extracting it to `utils`.